gateway/webhook/webhook.go

```diff
@@ -101,8 +101,17 @@ func parseWriteKey(req *http.Request) (writeKey string, found bool) {
 
 func (webhook *HandleT) failRequest(w http.ResponseWriter, r *http.Request, reason string, code int, stat string) {
 	writeKeyFailStats := make(map[string]int)
+	statTags := map[string]map[string]string{
+		stat: {
+			"reqType": "webhook",
+			`reason`:  reason,
```

Comment:
[optional] I'd use the backtick only if a) multiline is needed b) a double quote is needed in the string

```suggestion
			"reason":  reason,
```